src/librustc/hir/print.rs

```diff
@@ -1379,6 +1379,9 @@ impl<'a> State<'a> {
                 self.word_space(":")?;
                 self.print_type(&ty)?;
             }
+            hir::ExprKind::Use(ref expr) => {
+                self.print_expr(expr)?;
```

Comment:
this is not exactly right. I think you need to actually print the expanded version here

Reply:
I'll go with `{ let _t = $expr; _t }` then since it's the closest thing in surface Rust.